software/pkg-lists-wmla120.yml

```diff
@@ -632,82 +632,82 @@ yum_pkgs_p9:
-    - nvidia-driver-libs-418.29-4.el7.ppc64le.rpm
-    - nvidia-driver-NvFBCOpenGL-418.29-4.el7.ppc64le.rpm
-    - nvidia-driver-NVML-418.29-4.el7.ppc64le.rpm
-    - nvidia-libXNVCtrl-418.29-1.el7.ppc64le.rpm
-    - nvidia-libXNVCtrl-devel-418.29-1.el7.ppc64le.rpm
-    - nvidia-modprobe-418.29-1.el7.ppc64le.rpm
-    - nvidia-persistenced-418.29-1.el7.ppc64le.rpm
-    - nvidia-settings-418.29-1.el7.ppc64le.rpm
-    - nvidia-xconfig-418.29-1.el7.ppc64le.rpm
+    - cuda-10-1-10.1.105-1
+    - cuda-10.1.105-1
+    - cuda-command-line-tools-10-1-10.1.105-1
+    - cuda-compat-10-1-418.39-1
+    - cuda-compiler-10-1-10.1.105-1
+    - cuda-core-10-1-10.1.105-1
+    - cuda-cudart-10-1-10.1.105-1
+    - cuda-cudart-dev-10-1-10.1.105-1
+    - cuda-cufft-10-1-10.1.105-1
+    - cuda-cufft-dev-10-1-10.1.105-1
+    - cuda-cuobjdump-10-1-10.1.105-1
+    - cuda-cupti-10-1-10.1.105-1
+    - cuda-curand-10-1-10.1.105-1
+    - cuda-curand-dev-10-1-10.1.105-1
+    - cuda-cusolver-10-1-10.1.105-1
+    - cuda-cusolver-dev-10-1-10.1.105-1
+    - cuda-cusparse-10-1-10.1.105-1
+    - cuda-cusparse-dev-10-1-10.1.105-1
+    - cuda-demo-suite-10-1-10.1.105-1
+    - cuda-documentation-10-1-10.1.105-1
+    - cuda-driver-dev-10-1-10.1.105-1
+    - cuda-drivers-418.39-1
+    - cuda-gdb-10-1-10.1.105-1
+    - cuda-gdb-src-10-1-10.1.105-1
+    - cuda-gpu-library-advisor-10-1-10.1.105-1
+    - cuda-libraries-10-1-10.1.105-1
+    - cuda-libraries-dev-10-1-10.1.105-1
+    - cuda-license-10-1-10.1.105-1
+    - cuda-memcheck-10-1-10.1.105-1
+    - cuda-minimal-build-10-1-10.1.105-1
+    - cuda-misc-headers-10-1-10.1.105-1
+    - cuda-npp-10-1-10.1.105-1
+    - cuda-npp-dev-10-1-10.1.105-1
+    - cuda-nsight-10-1-10.1.105-1
+    - cuda-nsight-compute-10-1-10.1.105-1
+    - cuda-nsight-systems-10-1-10.1.105-1
+    - cuda-nvcc-10-1-10.1.105-1
+    - cuda-nvdisasm-10-1-10.1.105-1
+    - cuda-nvgraph-10-1-10.1.105-1
+    - cuda-nvgraph-dev-10-1-10.1.105-1
+    - cuda-nvjpeg-10-1-10.1.105-1
+    - cuda-nvjpeg-dev-10-1-10.1.105-1
+    - cuda-nvml-dev-10-1-10.1.105-1
+    - cuda-nvprof-10-1-10.1.105-1
+    - cuda-nvprune-10-1-10.1.105-1
+    - cuda-nvrtc-10-1-10.1.105-1
+    - cuda-nvrtc-dev-10-1-10.1.105-1
+    - cuda-nvtx-10-1-10.1.105-1
+    - cuda-nvvp-10-1-10.1.105-1
+    - cuda-runtime-10-1-10.1.105-1
+    - cuda-samples-10-1-10.1.105-1
+    - cuda-sanitizer-api-10-1-10.1.105-1
+    - cuda-toolkit-10-1-10.1.105-1
+    - cuda-tools-10-1-10.1.105-1
+    - cuda-visual-tools-10-1-10.1.105-1
+    - dkms-nvidia-418.39-1.el7
+    - libcublas10-10.1.0.105-1
+    - libcublas-devel-10.1.0.105-1
+    - libglvnd-1.0.1-0.8.git5baa1e5.el7
+    - libglvnd-core-devel-1.0.1-0.8.git5baa1e5.el7
+    - libglvnd-debuginfo-1.0.1-0.6.git5baa1e5.el7
+    - libglvnd-devel-1.0.1-0.8.git5baa1e5.el7
+    - libglvnd-egl-1.0.1-0.8.git5baa1e5.el7
+    - libglvnd-gles-1.0.1-0.8.git5baa1e5.el7
+    - libglvnd-glx-1.0.1-0.8.git5baa1e5.el7
+    - libglvnd-opengl-1.0.1-0.8.git5baa1e5.el7
+    - nvidia-driver-418.39-4.el7
```

Comment:
I want to reiterate what we talked about in the meeting. These libraries can change. Although, they are currently in public repo ... they might change later on. A simple change and risk change would be to change this:
`libglvnd-core-devel-1.0.1-0.8.git5baa1e5.el7`
to
`libglvnd-core-devel`

To further remove some of your concerns of driver dependencies we can leave the drivers as is ...